nginx.conf

```diff
@@ -49,7 +49,7 @@ http {
 
     <% proxies.each do |path, route| %>
       location <%= path %> {
-        resolver 8.8.8.8 ipv6=off;
+        resolver 169.254.0.2 ipv6=off;
```

Comment:
Add a comment here?

Reply:
Also - do we need to update all the resolvers?

Reply:
> Also - do we need to update all the resolvers?

Seems like not.

Reply:
Does using the BOSH resolver mess up the eregs redirect? Shouldn't.

Reply:
> Does using the BOSH resolver mess up the eregs redirect? Shouldn't.

Seems ok when everything is in place https://dev.fec.gov/regulations

Reply:
> Also - do we need to update all the resolvers?

Should be just for the cms application because that will be in an internal route. Other paths can use the 8.8.8.8 (Google) resolvers because they're all public.

Reply:
Comment added to original commit